gomod-sync/README.md

```diff
@@ -0,0 +1,101 @@
+# gomod-sync
+
+Utility tool to check and update the Go version specified in the `go.mod` files of all exercises.
+It works by specifying the desired Go version for all the `go.mod` files to be in: the `check` command
+will verify if all `go.mod` files are in the desired version and the `update` command will update all
+`go.mod` files to have the desired Go version.
+
+Some exercises must have its `go.mod` specify a Go version that is different from the other exercise's `go.mod`.
+This is supported by the `exceptions` key of the configuration file, where an entry must exist for each exercise
+that must not have the default version.
+
+## Usage
+
+```
+  gomod-sync command [flags]
+
+Available Commands:
+  check       Checks if all go.mod files are in the target version
+  completion  generate the autocompletion script for the specified shell
```

Comment:
I don't understand what this does/means, can you help me? :thinking: :see_no_evil:

Reply:
Which part? The `completion` command?

Reply:
Which part? The `completion` command?

Reply:
Yes, the completion command.

Reply:
Most shells like bash or zsh allow users to provide auto completion files that tell the shell how to autocomplete custom commands. The idea here is that if you write `gomod-sync ` and then press tab, your shell will autocomplete with some random file you have in the current directory, because the shell doesn't know better. What you want is the shell to cycle the list of commands that are specific to `gomod-sync` when you write `gomod-sync ` and then press tab. But for the shell to be able to do this, it has to know the syntax for `gomod-sync`, the list of commands,  flags, etc. These completion files allow a user to teach their shell about `gomod-sync` so it provides auto completions that make sense for the command.

What the `completion` command does is generate one of these completion files for `gomod-sync`. A user can then plug this file into their shell to provide auto completion for `gomod-sync`.

That command is implemented automatically by cobra. I think we can disable it if needed.

Reply:
Thanks for the info!